domains/jobbot.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Discord job posting bot",
+    "repo": "https://github.com/kybrd12/jobbot",
+    "owner": {
+        "username": "kybrd12",
+        "email": "knowledgebase.biolink.design@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://kybrd12.github.io/jobbot/"
```

```suggestion
        "CNAME": "kybrd12.github.io"
```
CNAME records cannot contain protocols (`https://`) or slashes (`/`).